apps/api/src/app/change/usecases/promote-notification-template-change/promote-notification-template-change.ts

```diff
@@ -154,7 +156,12 @@ export class PromoteNotificationTemplateChange {
         preferenceSettings: newItem.preferenceSettings,
         steps,
         _notificationGroupId: notificationGroup._id,
+        isBlueprint: command.organizationId === this.getBlueprintOrganizationId(),
       }
     );
   }
+
+  private getBlueprintOrganizationId(): string {
+    return NotificationTemplateRepository.getBlueprintOrganizationId();
+  }
```

Comment:
No need to modify, but in the future could be a great use of a getter:
```suggestion
  private get blueprintOrganizationId(): string {
    return NotificationTemplateRepository.getBlueprintOrganizationId();
  }
```